src/main.js

```diff
@@ -56,8 +56,8 @@ class LogClient {
             return;
         }
         const loggerUrl = isProduction() || this.environment === "prod"
-            ? "https://frontend-logger.cloud.nsb.no/log"
-            : "https://test.frontend-logger.cloud.nsb.no/log";
+            ? "https://frontend-logger.cloud.vy.no/log"
+            : "https://test.frontend-logger.cloud.vy.no/log";
         return fetch(loggerUrl, {
```

Comment:
Ser forbedringen din @knutbekk  og høyner med å gjøre kallet mot `www.vy.no/services/frontend-logger` istedenfor. Da kan vi bruke relativ URL i fetch, altså `fetch('/services/frontend-logger', ...)`. Da får vi DataDome-beskyttelse og vi kan kvitte oss med API Gateway-en til frontend-loggeren på sikt.

Reply:
Uh, det er jo fantastisk! Da slipper den å være så generisk for Vy også. Jeg kan endre jeg. Da vil det funke for testmiljøene også ut av boksen da, tipper jeg?

Reply:
Stemmer :)